### PR TITLE
Add docker images and helm chart

### PR DESCRIPTION
**Pull request by Freshchris01**
This PR adds:

- The docker image for spark and livy including a build script `acrbuild.sh` for each image
- A helm chart for livy
- Instructions on how to deploy the chart with helm
- Instructions on how to use the livy deployment with an interactive session

### CODE REVIEW

**Comment by edwin-huber** [comment]
Hi @Freshchris01 
I see that some of the files are using Apache License: https://www.apache.org/licenses/LICENSE-2.0
Have you clearly indicated any files that you have modified?

File: README.md [hunk 1]
```diff
@@ -26,6 +26,44 @@ Useful info:
   
 - https://spark.apache.org/docs/latest/running-on-kubernetes.html
   
+- Livy API documentation: https://livy.incubator.apache.org/docs/latest/rest-api.html
+
+1. Build the spark and the livy docker image by filling in your docker registry in the `acrbuild.sh` script in the `DockerImage/spark` as well as `DockerImage/livy` image
```
Comment (edwin-huber) [inline README.md:31]: We should ensure that all instances of <your-container-registry> are replaced in the yaml files.

**Review by edwin-huber — CHANGES_REQUESTED** [review]

**Comment by Freshchris01** [comment]
> Hi @Freshchris01 I see that some of the files are using Apache License: https://www.apache.org/licenses/LICENSE-2.0 Have you clearly indicated any files that you have modified?

I added a `NOTICE` file indicating and referencing the source of the `Dockerfile`

File: README.md (hunk 1 above)
Comment (Freshchris01) [reply README.md:31]: I replaced the image names with `<your-docker-registry>` and searched for occurrences of the old image. Didn't find more of them.

File: DockerImage/spark/NOTICE [hunk 2]
```diff
@@ -1 +1,4 @@
-The Dockerfile in this directoy was modified by a contributor of this repository (Christian Diemers) in the extend of changing the download link of the spark and hadoop binaries, because the old link is not reachable anymore. For the original version of this file, please check https://github.com/anildwarepo/spark-on-aks/blob/b7888394b62d9980ff7265b96ee289ed031cf9d7/spark/Dockerfile.
\ No newline at end of file
+This file contains changes to files in this directory, taken from https://github.com/anildwarepo/spark-on-aks/blob/b7888394b62d9980ff7265b96ee289ed031cf9d7/spark/.
```
Comment (Freshchris01) [inline DockerImage/spark/NOTICE:1]: This file includes the exact file changes.

**Review by edwin-huber — APPROVED** [review]

**Pull request merged**